CHANGELOG.md

```diff
@@ -23,6 +23,33 @@ This change was made in [pull request #3599: Enable new link styles by default](
 
 ### Breaking changes
 
+#### "Compatibility mode" features are no longer supported
+
+GOV.UK Frontend no longer supports compatibility with legacy frameworks. You can no longer incrementally add GOV.UK Frontend to an existing service using a previous GOV.UK framework. We have removed the following Sass variables and one mixin which controlled compatibility mode:
+
+- `$govuk-compatibility-govukelements`
+- `$govuk-compatibility-govuktemplate`
+- `$govuk-compatibility-govukfrontendtoolkit`
+- The `compatibility-mode` mixin which automatically checked if any of the 3 control variables were set to `true`
+
+We have additionally removed features that were managed via the above variables. The following features and their corresponding variables have now been removed:
```

Comment:
```suggestion
We have additionally removed features that were managed using the above variables. The following features and their corresponding variables have now been removed:
```

Reply:
not using 'via' is a style/accessibility thing - however I see no problem in keeping it, if using it provides the most accurate sentence.